skimage/conftest.py

```diff
@@ -8,15 +8,11 @@
     np.set_printoptions(legacy='1.13')
 
 # List of files that pytest should ignore
-collect_ignore = ["setup.py",
-                  "skimage/io/_plugins",
-                  "doc/",
-                  "tools/",
-                  "viewer_examples"]
+collect_ignore = ["io/_plugins",]
```

Comment:
Moving this file is pretty genius. =D

Reply:
Windows still can't use it. doctests on windows are still broken, even after that change.